app/dashboard/tasks.py

```diff
@@ -135,3 +136,21 @@ def m2m_changed_interested(self, bounty_pk, retry: bool = True) -> None:
         from dashboard.notifications import maybe_market_to_github
         maybe_market_to_github(bounty, 'work_started',
                                profile_pairs=bounty.profile_pairs)
+
+
+
+@app.shared_task(bind=True, max_retries=1)
+def increment_view_count(self, pks, content_type, retry: bool = True) -> None:
+    """
+    :param self:
+    :param pk:
+    :param content_type:
+    :return:
+    """
+    redis = RedisService().redis
```

Comment:
A good opportunity here to optimize performance is to swap out the `redis_service.py` code we use for just using a proper connection pool